src/Take.Blip.Builder/ContextExtensions.cs

```diff
@@ -25,15 +30,33 @@ public static Ticket GetTicket(this IContext context)
         public static void SetTicket(this IContext context, Ticket contact) 
             => SetValue(context, TICKET_KEY, contact);
 
+        public static void RemoveTicket(this IContext context)
+            => RemoveValue(context, TICKET_KEY);
+
+        public static async Task<TicketContext> GetTicketVariableAsync(this IContext context, CancellationToken cancellationToken)
+        {
+            var ticketJson = await GetVariableAsync(context, TICKET_KEY, cancellationToken);
+            if (string.IsNullOrEmpty(ticketJson))
+                return null;
+
+            return ParseFromJson<TicketContext>(ticketJson);
+        }
+
+        public static void SetTicketVariable(this IContext context, TicketContext ticketContext, CancellationToken cancellationToken)
```

Comment:
The class already only for context. I think it redundant.

Reply:
Yes, this is an extensions method class for IContext, all method interact with that. I've suggested that change because there is a method with the name `SetTicket` that sets a variable of `Ticket` type to the context. This one sets a variable of `TicketContext` type to the context. The name suggested could help to differentiate both methods